src/Analyzers/Core/Analyzers/DiagnosticCustomTags.cs

```diff
@@ -16,9 +15,12 @@ internal static class DiagnosticCustomTags
         /// <summary>
         /// it is string[] because DiagnosticDescriptor expects string[]. 
         /// </summary>
+        private static readonly string s_enforceOnBuildNeverTag = EnforceOnBuild.Never.ToCustomTag();
         private static readonly string[] s_microsoftCustomTags = new string[] { WellKnownDiagnosticTags.Telemetry };
-        private static readonly string[] s_editAndContinueCustomTags = new string[] { WellKnownDiagnosticTags.EditAndContinue, WellKnownDiagnosticTags.Telemetry, WellKnownDiagnosticTags.NotConfigurable };
+        private static readonly string[] s_editAndContinueCustomTags = new string[] { WellKnownDiagnosticTags.EditAndContinue, WellKnownDiagnosticTags.Telemetry, WellKnownDiagnosticTags.NotConfigurable, s_enforceOnBuildNeverTag };
         private static readonly string[] s_unnecessaryCustomTags = new string[] { WellKnownDiagnosticTags.Unnecessary, WellKnownDiagnosticTags.Telemetry };
+        private static readonly string[] s_notConfigurableCustomTags = new string[] { WellKnownDiagnosticTags.NotConfigurable, s_enforceOnBuildNeverTag, WellKnownDiagnosticTags.Telemetry };
```

Comment:
📝 Note to self: need to look up the exact semantics of `EnforceOnBuild.Never`

Reply:
@sharwell The EnforcedOnBuild custom tags are used by the post-build tool being added here: https://github.com/dotnet/roslyn/pull/49967/files#diff-0b599b648a408b6bb838bc4dc8019b2f9c0b263181d88f67cc347944f5410531R153-R187